Development/nmos/node_api_target_handler.h

```diff
@@ -25,18 +26,18 @@ namespace nmos
 
     // implement the Node API /receivers/{receiverId}/target endpoint using the Connection API implementation with the specified transport file parser and the specified validator
     // (the /target endpoint is only required to support RTP transport, other transport types use the Connection API)
-    node_api_target_handler make_node_api_target_handler(nmos::node_model& model, transport_file_parser parse_transport_file, details::connection_resource_patch_validator validate_merged);
+    node_api_target_handler make_node_api_target_handler(nmos::node_model& model, load_cacert_handler load_cacert, transport_file_parser parse_transport_file, details::connection_resource_patch_validator validate_merged);
```

Comment:
Did you consider backwards compatibility of these functions? Perhaps there's no need to keep the existing overloads if the majority of users rely on the call-site in `make_node_server`.

Reply:
Yes, I would think users will rely on the `make_node_server`, I will remove the existing overloads as your suggestion.

Reply:
do you mean like these?

```
namespace nmos
{
    struct node_model;

    // handler for the Node API /receivers/{receiverId}/target endpoint
    typedef std::function<pplx::task<void>(const nmos::id& receiver_id, const web::json::value& sender_data, slog::base_gate& gate)> node_api_target_handler;

    // implement the Node API /receivers/{receiverId}/target endpoint using the Connection API implementation with the specified transport file parser and the specified validator
    // (the /target endpoint is only required to support RTP transport, other transport types use the Connection API)
    node_api_target_handler make_node_api_target_handler(nmos::node_model& model, load_cacert_handler load_cacert, transport_file_parser parse_transport_file, details::connection_resource_patch_validator validate_merged);

    // implement the Node API /receivers/{receiverId}/target endpoint using the Connection API implementation with the specified transport file parser and the specified validator
    // (the /target endpoint is only required to support RTP transport, other transport types use the Connection API)
    inline node_api_target_handler make_node_api_target_handler(nmos::node_model& model, transport_file_parser parse_transport_file, details::connection_resource_patch_validator validate_merged)
    {
        return make_node_api_target_handler(model, {}, parse_transport_file, validate_merged);
    }

    // implement the Node API /receivers/{receiverId}/target endpoint using the Connection API implementation with the specified transport file parser
    // (the /target endpoint is only required to support RTP transport, other transport types use the Connection API)
    inline node_api_target_handler make_node_api_target_handler(nmos::node_model& model, transport_file_parser parse_transport_file)
    {
        return make_node_api_target_handler(model, {}, parse_transport_file, {});
    }

    // implement the Node API /receivers/{receiverId}/target endpoint using the Connection API implementation with the default transport file parser
    // (the /target endpoint is only required to support RTP transport, other transport types use the Connection API)
    node_api_target_handler make_node_api_target_handler(nmos::node_model& model, load_cacert_handler load_cacert);
}

```